tests/Tests/800__ocm/800__ocm.robot

```diff
@@ -0,0 +1,35 @@
+*** Settings ***
+Library         SeleniumLibrary
+Resource        ../../Resources/Page/ODH/ODHDashboard/ODHDashboard.resource
+Resource        ../../Resources/Page/HybridCloudConsole/HCCLogin.robot
+Suite Setup     OCM Test Setup
+Suite Teardown  OCM Test Teardown
+
+*** Test Cases ***
+Verify User Can Access RHODS Documentation
```

Comment:
Title could be `Verify User Can Access RHODS Documentation From OCM Console` or similar to give more context